BundleTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿using NUnit.Framework;
+
+namespace Task1.UnitTests
+{
+    [TestFixture]
+    public class AngleTests
+    {
+        [Test]
+        public void ConstructorTest()
+        {
+            var bundle = new Bundle(5, 10);
+            Assert.That(bundle.BanknoteType, Is.EqualTo(5));
+            Assert.That(bundle.Count, Is.EqualTo(10));
+            
+        }
+
+        [TestCase(-1)]
+        public void NegativeCountTest_ArgumentException(int val)
+        {
+            var bundle = new Bundle();
+            Assert.That(() => bundle.Count = val, Throws.ArgumentException);
+        }
+        
+        [TestCase(18)]
+        public void BanknoteTest_ArgumentException(int val)
+        {
+            var bundle = new Bundle();
+            Assert.That(() => bundle.BanknoteType = val, Throws.ArgumentException);
+        }
```

Comment:
Не имеет смысла использовать параметрический тест, если вы тестируете только один случай.
```suggestion
        [TestCase]
        public void NegativeCountTest_ArgumentException()
        {
            var bundle = new Bundle();
            Assert.That(() => bundle.Count = -1, Throws.ArgumentException);
        }
        
        [Test]
        public void BanknoteTest_ArgumentException()
        {
            var bundle = new Bundle();
            Assert.That(() => bundle.BanknoteType = 18, Throws.ArgumentException);
        }
```